### DISCUSSION TITLE
Version 0.6.0 Alpha 5

### DISCUSSION THREAD

**Discussion by DSheirer** [DSheirer/sdrtrunk#1517] [Announcements]
## 1. Features:
* SDRPlay RSP tuner support 

Note: SDRPlay RSP tuner support requires SDRPlay API version 3.07 - 3.11

## 2. Requirements:
* Windows x86 (64-bit)
* Linux x86/aarch64 (64-bit)
* OSX x86/aarch64 (64-bit)

**Note**: JMBE library must be manually downloaded/built for OSX aarch64 due to OSX sandboxing of downloaded code.
**Note**: Linux for arm-32 is NOT currently supported

## 3. Optional Requirements
**JMBE version 1.0.9+** is required for decoding APCO25 Phase 1/2 and DMR digital audio on Linux and OSX aarch64 and this library can be compiled within the sdrtrunk application.

Legacy instructions for compiling the JMBE library are located at: **https://github.com/DSheirer/sdrtrunk/wiki/JMBE**

## 4. User Manual
* [Wiki](https://github.com/DSheirer/sdrtrunk/wiki)
* [User Manual](https://github.com/DSheirer/sdrtrunk/wiki/User-Manual)

## 5. Support
* [Support](https://github.com/DSheirer/sdrtrunk/wiki/Support)

## 6. What's Changed
* #52 SDRPlay RSP tuner support. by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1453
* #1405 Thread Leak in Polyphase Channelizer by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1477
* #1456 Shutdown Error When SDRPlay API Not Installed by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1457
* #1459 Gradle Doesn't Produce Correct OS Releases in Version 0.6.0 by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1463
* #1461 SDRPlay Library Path on OSX by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1462
* #1464 Reducing Logging When SDRPlay API Not Installed On Local Machine by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1465
* #1471 Enhance Processing of Unknown Hytera packet messages. by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1483
* #1473 RSP Tuners - Provide IF AGC Control & Power Overload Indicators by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1484
* #1485 Sets Blackman-Harris 7 As Default FFT Window by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1486
* #1487 Thread Locking Impacting Processing Workload by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1488
* #1489 SDRPlay RSP Tuners - Sample Rate Control Lock/Unlocking on Channel Sourcing by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1490
* #1491 Polyphase Channellizer - Add Diagnostic Logging to Dispatcher Thread by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1492
* #1498 JMBE Audio Class Loading - Memory Leak by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1499
* #1500 Channel Auto-Start Timeout As a User Preference by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1501
* #1506 Resolve App Lockup, Tuner Freezes, Buffer Overflows & Excessive Memory Usage by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1513
* #1511 SDRPlay API Crashes App on Startup When Linux Service is Stopped by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1514
* #1515 Updates version to 0.6.0 alpha 5 by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1516

**Full Changelog**: https://github.com/DSheirer/sdrtrunk/compare/v0.5.1...v0.6.0-alpha.5

<hr /><em>This discussion was created from the release <a href='https://github.com/DSheirer/sdrtrunk/releases/tag/v0.6.0-alpha.5'>Version 0.6.0 Alpha 5</a>.</em>

**Comment by louisik1** [comment]
Thanks!

I'm using 0.5.0.  Can I download this 0.6.0.a5 and just switch to it, or are there any breaking changes in user settings or anything like that that would require me to re-configure my system?  And if I switch to it and find blocking issues for me, can I simply switch back to 0.5.0 without problems, i.e. 0.6.0.a5 making changes to the configuration files that 0.5.0 might choke on?

edit: 0.6.0.a7..  whatever the latest is. :)
